src/scrcmd.c

```diff
@@ -3134,6 +3178,21 @@ bool8 ScrFunc_hidefollower(struct ScriptContext *ctx)
     return TRUE;
 }
 
+static bool8 ScriptContext_NextCommandEndsScript(struct ScriptContext * ctx)
+{
+    const u8 * script = ctx->scriptPtr;
+    u8 nextCmd = *script;
+    if (nextCmd == 3) // return
+    {
+        script = ctx->stack[ctx->stackDepth - 1];
+        nextCmd = *script;
+    }
+    if (nextCmd < 0x6B || nextCmd > 0x6C) // releaseall or release
+        return FALSE;
+    else
+        return TRUE;
+}
+
```

Comment:
```suggestion
static bool8 ScriptContext_NextCommandEndsScript(struct ScriptContext * ctx)
{
    if (!OW_ALLOW_INSTANT_WALKAWAY)
        return FALSE;

    const u8 * script = ctx->scriptPtr;
    u8 nextCmd = *script;
    if (nextCmd == 3) // return
    {
        script = ctx->stack[ctx->stackDepth - 1];
        nextCmd = *script;
    }
    if (nextCmd < 0x6B || nextCmd > 0x6C) // releaseall or release
        return FALSE;
    else
        return TRUE;
}

```

We can argue about this feature being default on or off (I think off because SV doesn't have it, given our current guidelines) BUT I do think there needs to be a way to opt out with using that special in every single script. Could you please add a `OW_ALLOW_INSTANT_WALKAWAY` to `/include/config/overworld.h`?